pom.xml

```diff
@@ -519,6 +519,27 @@
         <version>${plugin.deploy.version}</version>
       </plugin>
 
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
```

Comment:
AFAIK, it doesn't need to update LICENSE or LICENSE_BIN if it's not included in binary. @Leemoonsoo Can you verify it?